src/settings_gui.cpp

```diff
@@ -170,6 +170,7 @@ struct GameOptionsWindow : Window {
 	GameSettings *opt;
 	bool reload;
 	int gui_scale;
+	static int active_tab;
```

Comment:
```suggestion
	static inline int active_tab = WID_GO_TAB_GENERAL;
```
and remove the definition after the class?